### PR TITLE
Rubbish and blood drip sound effects

### PR DESCRIPTION
**Pull request by cacogen**
:cl: cacogen
soundadd: Empty junk food bags crinkle when walked on
soundadd: Blood droplets drip when they hit the ground
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Rubbish sound effect is just part of my desire for there to be more sounds in the game, and while the droplets serve the same purpose they also draw more attention to bleeding which can be missed.

Drip source: https://freesound.org/people/Marth8880/sounds/380375/
Crinkling source: https://freesound.org/people/deleted_user_7146007/sounds/383451/
Puddle drip source: https://freesound.org/people/davidferoli/sounds/190661/
Blood puddle splash source: https://freesound.org/people/Slave2theLight/sounds/157113/

[blood_drip1.ogg](https://clyp.it/5ttzln0l)
[blood_drip2.ogg](https://clyp.it/roaupitl)
[blood_drip3.ogg](https://clyp.it/hkcg4fbu)
[blood_drip_puddle.ogg](https://clyp.it/mawjf3eg)
[blood_puddle_splash.ogg](https://clyp.it/akfggc25)

[trash_step1.ogg](https://clyp.it/k3lhdnqu)
[trash_step2.ogg](https://clyp.it/o0dgbqd1)
[trash_step3.ogg](https://clyp.it/swnaw40p)

- [x] Add source metadata to new .oggs
- [x] Push new sounds
- [x] Upload new sounds to clyp.it for people to listen to

### CODE REVIEW

**Comment by chesse20** [comment]
press the merge button right now and add this!

**Comment by Vidiveni** [comment]
I like the trash sounds, but I think the blood drips sound a bit too similar to broken headphone/broken audio driver crackle.

**Comment by cacogen** [comment]
It was really hard to find something that sounded like a drop hitting a hard surface. Most of the sounds had an echo or were hitting water/metal/something else. I can look a little harder though.

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -6,34 +6,47 @@
 	desc = "This is rubbish."
 	w_class = WEIGHT_CLASS_TINY
 	resistance_flags = FLAMMABLE
+	var/step_sound = null
+
+/obj/item/trash/Crossed(mob/living/L)
+	if(istype(L) && has_gravity(loc))
+		playsound(loc, step_sound, L.has_trait(TRAIT_LIGHT_STEP) ? 20 : 50, 1)
```
Comment (AnturK) [inline code/game/objects/items/trash.dm]: playsound(src,

File: code/modules/mob/living/blood.dm [hunk 2]
```diff
@@ -97,8 +97,14 @@
 		if(isturf(src.loc)) //Blood loss still happens in locker, floor stays clean
 			if(amt >= 10)
 				add_splatter_floor(src.loc)
+				playsound(loc, pick('sound/effects/blood_puddle_splash.ogg'), 30, 1)
```
Comment (AnturK) [inline code/modules/mob/living/blood.dm]: playsound(src, again, and in all other places not going to repeat.

You're also calling pick on a  single element here and below, no need for that.

**Review by AnturK — CHANGES_REQUESTED** [review]
Few code issues.

Also might get bit annoying without any timeouts but that's less important.

File: code/modules/mob/living/blood.dm (hunk 2 above)
Comment (cacogen) [reply code/modules/mob/living/blood.dm]: thanks

File: code/game/objects/items/trash.dm (hunk 1 above)
Comment (cacogen) [reply code/game/objects/items/trash.dm]: thanks

**Comment by cacogen** [comment]
@AnturK 
Okay, I've made the changes you requested and squashed the commits.

**Comment by ShizCalev** [comment]
Trash step sounds have alot of background hum to them.

puddle splash sound isn't properly terminated (end of file should be zeroed out, it's at -18db.) this will lead to audio crackling.

**Comment by Mickyan** [comment]
It wouldn't work when I tried to test it for some reason, what's the hearing range on the blood drip? Ideally you should only hear it if you're in immediate proximity or it'll get really annoying

**Comment by cacogen** [comment]
I'm gonna close this for now because I can't be assed working on it any further. Thanks for the help and feedback everyone.

**Comment by chesse20** [comment]
what its already finished

**Pull request closed without merging**